src/lib/Libutil/pbs_aes_encrypt.c

```diff
@@ -237,6 +242,10 @@ decode_from_base64(char* buffer, unsigned char** ret_decoded_data, size_t* ret_d
 
 	mem_obj1 = BIO_new_mem_buf(buffer, -1);
 	mem_obj2 = BIO_new(BIO_f_base64());
+	if (mem_obj2 == NULL) {
+		BIO_free_all(mem_obj1);
+		return (1);
+	}
 
 	mem_obj1 = BIO_push(mem_obj2, mem_obj1);
```

Comment:
check for return of BIO_push()

Reply:
nowhere documented that it returns something in case of failure